libexec/bats-exec-test

```diff
@@ -68,6 +71,10 @@ load() {
   source "${filename}"
 }
 
+# timeout from here: http://stackoverflow.com/questions/3504945/timeout-command-on-mac-os-x
+# in case 'timeout' command does not exist on the system
+function timeout() { perl -e 'alarm shift; exec @ARGV' "$@"; }
```

Comment:
Oh, right. I had put that in there because I was working locally and Mac doesn't have the timeout command, but all the Linux machines should.